gef.py

```diff
@@ -5467,8 +5475,9 @@ def do_invoke(self, argv: List) -> None:
         info(f"Listing custom structures from '{manager.path}'")
         struct_color = gef.config["pcustom.structure_type"]
         filename_color = gef.config["pcustom.structure_name"]
-        for module in manager.modules:
-            __modules = ", ".join([Color.colorify(structure.name, struct_color) for structure in module.structures])
+        for module_name in manager.modules:
```

Comment:
`for module in manager.modules.values():`

Reply:
not a dict

Reply:
Is now?